### PR TITLE
COM-19972: Implement 'Article' full view

### PR DESCRIPTION
**Pull request by clash82**
https://jira.ez.no/browse/COM-19972

**Please do not merge until `1.8` will be released.**
Requires this patch https://github.com/ezsystems/ezpublish-kernel/pull/1860 to work.

Screenshot:

screen shot 2016-12-08 at 15 38 54

### CODE REVIEW

**Comment by clash82** [comment]
ping @damianz5

File: app/Resources/views/full/article.html.twig [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{% block main %}
+    <main class="white-background">
+{% endblock %}
+
+{% block content %}
+    <div class="container article">
+        <h1>{{ ez_content_name(content) }}</h1>
+
+        <div class="article-content">
+            {{ ez_render_field(content, 'body') }}
```
Comment (SylvainGuittard) [inline app/Resources/views/full/article.html.twig]: ```intro``` field is missing.

I also recommend to have all the fields in a dedicated ```<div>```
```
 <div class="article-content">
    <div class="article-body"> 
        {{ ez_render_field(content, 'body') }}
   </div>
</div>
```

Comment (clash82) [inline app/Resources/views/full/article.html.twig]: AFAIK, intro field was used only for line views??? @SylvainGuittard r u sure?

Comment (SylvainGuittard) [inline app/Resources/views/full/article.html.twig]: Nothing is defined on ezplatform.com. I will suggest to include it for now.

Comment (clash82) [inline app/Resources/views/full/article.html.twig]: Added.

**Comment by SylvainGuittard** [comment]
Thanks @clash82

**Pull request merged**